webapp/public/index.html

```diff
@@ -22,6 +22,10 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
+    <!-- React Bootstrap -->
+    <!-- Latest compiled and minified CSS -->
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u" crossorigin="anonymous">
```

Comment:
Why not Bootstrap 4? Latest version is 4.2.1

<link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.2.1/css/bootstrap.min.css" integrity="sha384-GJzZqFGwb1QTTN6wy59ffF1BuGJpLSa9DkKMp0DgiMDm4iYMj70gZWKYbI706tWS" crossorigin="anonymous">

Reply:
Oh nevermind, I see it's because react-bootstrap only supports 3.X

Reply:
Yeah so the react-bootstrap say they are working on bootstrap 4 support, but that will probably take a while.

We do have the option of not using react-bootstrap and using plain bootstrap 4, but that we cant use the react-bootstrap components or some of its features. 

It is basically 
```
// React-Bootstrap
<NavItem eventKey="1" href="/home">
          NavItem 1 content
</NavItem>
```
vs
```
// Normal Bootsrap
<li className="nav-item">
      <a href="/home className="nav-link"> NavItem 1 content </a>
 </li>
```

We can also change at anytime, but react-bootstrap makes life easier for people who aren't comfortable with html + css.

I am comfortable either way, what do you two think? @avishkar58 @jaderabbit